src/enlyze/api_clients/production_runs/models.py

```diff
@@ -0,0 +1,115 @@
+from abc import abstractmethod
+from datetime import datetime, timedelta
+from typing import Any, Optional
+from uuid import UUID
+
+import enlyze.models as user_models
+from enlyze.api_clients.base import ApiBaseModel
+
+
+class ProductionRunsApiModel(ApiBaseModel):
+    """Base class for Production Runs API object models using pydantic
+
+    All objects received from the Production Runs API are passed into models that derive
+    from this class and thus use pydantic for schema definition and validation.
+
+    """
+
+    @abstractmethod
+    def to_user_model(self, *args: Any, **kwargs: Any) -> Any:
+        pass
```

Comment:
This line reports as missing in the coverage report. Please fix.

Reply:
How would you test this? Because this is an abstract method, you can't even instantiate the class :thinking:

Reply:
As you say, it's an abstract method, so it can't be tested. But you can do either:

```suggestion
        pass # pragma: no cover
```

or

```suggestion
        """Some useful docstring"""
```

Reply:
```suggestion
        """Base model for Production Runs API response models"""
```

Reply:
Not really, I mean this is a function, so one usually describes its behaviour. Rather something like

```suggestion
        """Convert to a model that will be returned to the user."""
```

Reply:
Sorry I wasn't thinking clearly